tests/kyber.rs

```diff
@@ -3,16 +3,14 @@ use libcrux::{
     kem::{self, Algorithm, Ct, PrivateKey},
 };
 
+use rand::{CryptoRng, Rng};
+
 #[cfg(not(target_arch = "wasm32"))]
-use libcrux::drbg::{self, RngCore};
+use libcrux::drbg;
 #[cfg(target_arch = "wasm32")]
 use rand_core::{OsRng, RngCore};
 
```

Comment:
`RngCore` here is unused now.

```suggestion
use rand_core::OsRng;
```